src/components/spaces/show/index.js

```diff
@@ -106,9 +108,12 @@ export default class SpacesShow extends Component {
             <div className='col-sm-10'>
 
               {space &&
-                <SpacesShowHeader onDestroy={this.destroy.bind(this)}
+                <SpacesShowHeader
+                    isLoggedIn={e.me.isLoggedIn(this.props.me)}
```

Comment:
I'd probably move this logic above the ``return`` method, but it's a very minor issue.